model/Mastermind.php

```diff
@@ -174,4 +174,13 @@ public function addProposition(Combination $proposition)
         $this->propositions[] = $proposition;
     }
 
+    /**
+     * Décrémente le nombre d'essais restants
+     * @return void
+     */
+    public function decrementRemainingAttempts(): void
```

Comment:
Je pense qu'il serait mieux de créer une méthode : 
```php
Mastermind::play(Combinaison $proposition): void
```
Cette méthode permettrait de proposer une combinaison, tu ferais dans cette fonction les "petites actions" comme `$this->remainingAttempts--` ou l'ajout ds le tableau des compareResults ...

Tu peux aussi imaginer une méthode `Mastermind::isGameOver(): bool` qui renvoie `true` si la partie est perdue, `Mastermind::isGameWon(): bool` qui serait son pendant pour la partie gagnée.